src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/Address.java

```diff
@@ -29,4 +32,10 @@ public class Address {
     private String postTown;
     @JsonProperty("PostCode")
     private String postCode;
+
+    @JsonIgnore
+    public boolean isEmpty() {
+        return Stream.of(addressLine1, addressLine2, addressLine3, county, country, postTown, postCode)
```

Comment:
Nice. Of course Ashley would have done 
`.allMatch(StringUtils::isBlank);`

😉